base/utf8.jl

```diff
@@ -72,7 +72,7 @@ function next(s::UTF8String, i::Int)
         end
         if 0 < j && i <= j+utf8_trailing[d[j]+1] <= length(d)
             # b is a continuation byte of a valid UTF-8 character
```

Comment:
I know you didn't write this, but should this comment be referring to `d` instead of `b`?